platforms/access-bridge/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@jwp/access-bridge",
+  "version": "1.0.0",
+  "main": "build/main.js",
+  "private": true,
+  "author": "JW Player",
+  "type": "module",
+  "scripts": {
+    "build": "tsc && [ -f .env ] && cp .env ./build/.env || true && [ -f .env.local ] && cp .env.local ./build/.env.local || true",
```

Comment:
Did you look at using vite to do the node build? It would probably be useful to align with the rest of the web app code.